webrtc.html

```diff
@@ -1895,10 +1899,18 @@ <h4>Set the RTCSessionDescription</h4>
                     <var>track</var> to <var>stream</var>.</p>
                   </li>
                   <li>
-                    <p>For each <code><a>RTCTrackEvent</a></code>
-                    <var>trackEvent</var> in <var>trackEvents</var>,
-                    <a>fire a track event</a> named <code title=
-                    "event-track"><a>track</a></code> with <var>trackEvent</var>
+                    <p>For each entry <var>entry</var> in <var>trackEventInits</var>,
```

Comment:
nit: I don't know what "type" means for the constructor, but there is an RTCTrackEvent constructor taking the RTCTrackEventInit:
```
[Constructor(DOMString type, RTCTrackEventInit eventInitDict),
 Exposed=Window]
```
So maybe you could make this shorter by saying something like "Construct an RTCTrackEvent using the entry as the RTCTrackEventInit argument" and fire it. These steps are roughly redefining the constructor.

Reply:
That's a good idea!
Another idea I played with was to let the `trackEvents` continue to hold the events, and then here, instead of using `Fire an event` use `[dispatch](https://dom.spec.whatwg.org/#dispatching-events) event`. Not sure what way is best here.

Reply:
Instead of fire, you can use https://dom.spec.whatwg.org/#concept-event-create and https://dom.spec.whatwg.org/#concept-event-dispatch. I'd generally try to avoid that, but if that's the only thing that works here, that's the way to go. Please don't try invoking the constructor of things yourself as there's all kinds of complicated details there that really should be abstracted over across the whole web platform.

Reply:
Reading https://dom.spec.whatwg.org/#concept-event-create I stumble on the note "This does not work if members are required", and https://w3c.github.io/webrtc-pc/#dom-rtctrackeventinit requires receiver, track and transceiver.

The easiest way out seems to be to do what the PR in its current form does - though a bit ugly.